app/src/main/java/org/fossasia/openevent/general/settings/LegalFragment.kt

```diff
@@ -0,0 +1,77 @@
+package org.fossasia.openevent.general.settings
+
+import android.graphics.Color
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.support.v7.preference.Preference
+import android.view.*
+import com.takisoft.fix.support.v7.preference.PreferenceFragmentCompat
+import org.fossasia.openevent.general.R
+import org.fossasia.openevent.general.utils.Utils
+import java.util.prefs.PreferenceChangeEvent
+import java.util.prefs.PreferenceChangeListener
+
+class LegalFragment : PreferenceFragmentCompat(), PreferenceChangeListener {
+    val PRIVACY_LINK: String = "https://eventyay.com/privacy-policy/"
+    val TERMS_OF_SERVICE_LINK: String = "https://eventyay.com/terms/"
+    val COOKIE_POLICY_LINK: String = "https://eventyay.com/cookie-policy/"
+
+    override fun preferenceChange(evt: PreferenceChangeEvent?) {
+        preferenceChange(evt)
```

Comment:
`evt` is not a meaningful name, can you please change it to something meaningful